frontend/lib/norent/letter-builder/create-account.tsx

```diff
@@ -45,7 +45,7 @@ export const NorentCreateAccount = MiddleProgressStep((props) => {
               I agree to the{" "}
               <ModalLink
                 to={NorentRoutes.locale.letter.createAccountTermsModal}
-                component={PrivacyInfoModal}
+                component={() => <PrivacyInfoModal isForNorentSite />}
               >
```

Comment:
I had to do a wee bit of refactoring of the `PrivacyInfoModal` component, just to allow it to dynamically add a URL suffix to the outbound link for our new norent-specific policy. See below for my implementation, and let me know if there are any issues!

Reply:
Noice. Another way to do it is to look at `getGlobalAppServerInfo().siteType`, which is actually the function we use at the very beginning of the front-end code to know what site to render.  I kind of like the explicit-ness of your approach though.

Er actually do you mind changing `component` to `render`?  See my other comment in this review for an explanation of why.